lib/i18n_sonder/workers/sync_translations.rb

```diff
@@ -26,14 +26,10 @@ def sync(approved_translations_only:)
           translations_by_model_and_id = translation_result.success
           handle_failure(translation_result.failure)
 
-          begin
```

Comment:
moving the begin/rescue block to `write_translations` so we can catch individual failures and proceed.